### ISSUE TITLE
Check if we need to use BuiltIns in our autocompleter

### ISSUE THREAD

**Issue by jan-vcapgemini** [devonfw/IDEasy#168]
Check if code snippet should be used in Ide:
```
// Supplier<Path> workDir = context::getCwd;
// set up JLine built-in commands
// TODO: fix BuiltIns or remove
// Builtins builtins = new Builtins(workDir, null, null);
// builtins.rename(Builtins.Command.TTOP, "top");
// builtins.alias("zle", "widget");
// builtins.alias("bindkey", "keymap");
```

**Comment by hohwille** [comment]
See #56

**Comment by hohwille** [comment]
* we should create a `cd` command as feature in `ide shell` -> new ticket
* we should remove the TODO (and outcommented stuff?) from our code
* we should then close this ticket as invalid

**Comment by jan-vcapgemini** [comment]
Issue: #737 was created and PR #738 will remove the TODO. Can be closed after merge of #738.
